cli/README.md

```diff
@@ -232,15 +232,30 @@ It's important that the path you specify refers to the platform-specific, prepar
 | React Native wo/assets (iOS)     | `react-native bundle --platform ios --entry-file <entryFile> --bundle-output <bundleOutput> --dev false`                                                   | Value of the `--bundle-output` option                                                                 |
 | React Native w/assets (iOS)      | `react-native bundle --platform ios --entry-file <entryFile> --bundle-output <releaseFolder>/<bundleOutput> --assets-dest <releaseFolder> --dev false` | Value of the `--assets-dest` option, which should represent a newly created directory that includes your assets and JS bundle |
 
-### Target binary version parameter
+### Target binary range parameter
 
-This specifies the semver-compliant (e.g. `1.0.0` not `1.0`) store/binary version of the application you are releasing the update for. Only users running this **exact version** will receive the update. Users running an older and/or newer version of the app binary will not receive this update, for the following reasons:
+This specifies a [semver range expression](https://github.com/npm/node-semver#advanced-range-syntax) that covers all the store/binary versions that this update applies to. Any client device running a version of the binary that satisfies the range expression (i.e. `semver.satisfies(version, range)` returns `true`) will get the update. Examples of valid semver range expressions are as follows:
+
+| Range Expression | Who gets the update                                                                    |
+|------------------|----------------------------------------------------------------------------------------|
+| `1.2.3`          | Only devices running the specific binary app store version `1.2.3` of your app         |
+| `*`              | Any device configured to consume updates from your CodePush app                        |
+| `1.2.x`          | Devices running major version 1, minor version 2 and any patch version of your app     |
+| `1.2.3 - 1.2.7`  | Devices running any binary version between `1.2.3` (inclusive) and `1.2.7` (inclusive) |
+| `>=1.2.3 <1.2.7` | Devices running any binary version between `1.2.3` (inclusive) and `1.2.7` (exclusive) |
+| `~1.2.3`         | Equivalent to `>=1.2.3 <1.3.0`                                                         |
+| `^1.2.3`         | Equivalent to `>=1.2.3 <2.0.0`                                                         |
```

Comment:
I'd also like to understand how we behave with pre-release tags: https://github.com/npm/node-semver#prerelease-tags